content/doc/developer/plugin-development/choosing-jenkins-baseline.adoc

```diff
@@ -32,6 +32,9 @@ There are link:https://stats.jenkins.io/pluginversions/[statistics of core versi
 These are updated monthly.
 When updating the core dependency, choose the link:/changelog-stable/[newest LTS version] that doesn't exclude a majority of your existing users (by requiring a newer Jenkins than they have).
 
+NOTE:  if you are packaging a pure API library (one that does not depend on Jenkins APIs) then you should ignore newer jenkins versions and pick an older LTS.
+Something around 1 year old that does not have too many detached plugins makes a good choice and *PLACEHOLDER_OLDEST_LTS* would be a reasonable candidate.
```

Comment:
currently 2.277.4 as seen in https://deploy-preview-5220--jenkins-io-site-pr.netlify.app/doc/developer/plugin-development/choosing-jenkins-baseline/#choosing-a-version